README.md

```diff
@@ -11,16 +11,32 @@ This setup will perform the following operations:
 - install the Raspberry Pi kernel headers
 - install the required packages
 - compile the I2S drivers
+- update the asoundrc file
```

Comment:
Several lists in this file include this line.  I would like a _little_ information added to each one to mention the content of the different updates.